examples/aggregator/scripts/docker_push

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+readonly GLOBAL_SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")/../../../scripts/"
+readonly SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")"
+# shellcheck source=scripts/common
+source "${GLOBAL_SCRIPTS_DIR}/common"
+
+readonly AGGREGATOR_DOCKER_IMAGE_NAME='gcr.io/oak-ci/oak-aggregator'
+readonly BACKEND_DOCKER_IMAGE_NAME='gcr.io/oak-ci/oak-aggregator-backend'
+
+readonly IMAGES="$(find ./bazel-cache -name aggregator_image.tar)"
```

Comment:
This seems a very crude way to find those images. Can they not be found under a specific path?

Reply:
This path depends on the username:
```shell
"./bazel-cache/_bazel_${USERNAME}/b6bc0828d5b43c7085b650e0698bf388/execroot/oak/bazel-out/k8-dbg/bin/examples/aggregator/server/aggregator_image.tar"
```

Reply:
But why are they not under `bazel-out` or `bazel-bin` or some other variation?

Reply:
```shell
$ find . -name 'aggregator_*.tar'                                                                                     1 ↵ 
./bazel-cache/_bazel_ivanpetrov/b6bc0828d5b43c7085b650e0698bf388/execroot/oak/bazel-out/k8-dbg/bin/examples/aggregator/server/aggregator_image.tar
./bazel-cache/_bazel_ivanpetrov/b6bc0828d5b43c7085b650e0698bf388/execroot/oak/bazel-out/k8-dbg/bin/examples/aggregator/server/aggregator_image-layer.tar
./bazel-cache/_bazel_ivanpetrov/b6bc0828d5b43c7085b650e0698bf388/execroot/oak/bazel-out/k8-fastbuild/bin/examples/aggregator/server/aggregator_backend_image-layer.tar
./bazel-cache/_bazel_ivanpetrov/b6bc0828d5b43c7085b650e0698bf388/execroot/oak/bazel-out/k8-fastbuild/bin/examples/aggregator/server/aggregator_backend_image.tar
```

Reply:
`bazel-bin` is only available inside Docker.
And it's now only possible to push Docker image outside the dev Docker.

Reply:
I think it should be possible to copy things out from bazel-bin to another output directory, while running in Docker? That seems cleaner than just looking around in the bazel cache.

Reply:
Yes, I agree
So we need a new temporary directory for this. And we probably need to always check that this directory is cleared, once something like `bazel clean` is running.
Is it possible to use `target` directory for this? Or it's better to keep it only for `rust`?

Reply:
Perhaps go ahead with this PR as it is for now, but create a separate issue to track copying artefacts to an output directory (can just be called `out`, and git-ignored). We can then copy Rust binaries, wasm files, Docker images, Oak applications and anything else in there.

Reply:
Created an issue https://github.com/project-oak/oak/issues/719